branch_name_verification.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+import robot.basestation.app as app
+from robot.basestation.app import run_shell
+import re
+
+excluded_branches = {'master', 'develop', 'staging', 'test'}
+
+output, error = run_shell("git symbolic-ref --short HEAD", "", False)
```

Comment:
this function seems to be supposed to be used like this

`run_shell("git", "symbolic-ref --short HEAD", False)`

Reply:
Fixed